pyrobosim/examples/demo_astar_grid.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import os
+
+from pyrobosim.core.yaml import WorldYamlLoader
+from pyrobosim.utils.general import get_data_folder
+from pyrobosim.utils.pose import Pose
+from pyrobosim.navigation.astar_grid import AstarGrid
+
+# Load a test world.
+data_folder = get_data_folder()
+loader = WorldYamlLoader()
+w = loader.from_yaml(os.path.join(data_folder, "test_world.yaml"))
```

Comment:
Try use `world` for the variable name. I may have been guilty of doing this myself at some point, but still...